src/main/kotlin/lotto/LottoMatchType.kt

```diff
@@ -0,0 +1,17 @@
+package lotto
+
+enum class LottoMatchType(val matchCount: Int, val reward: String, val description: String, val hasBonus: Boolean) {
```

Comment:
코틀린 컨벤션에 따르면 생성파라미터가 여러개인경우 줄바꿈을 해주는게 좋다고 합니다!
```suggestion
enum class LottoMatchType(
    val matchCount: Int,
    val reward: String,
    val description: String,
    val hasBonus: Boolean,
) {
```